appinfo/info.xml

```diff
@@ -6,15 +6,15 @@
     <summary><![CDATA[Breeze Dark theme for Nextcloud, based on the theme by KDE]]></summary>
     <description><![CDATA[
 # Breeze Dark
-This is a breeze dark theme for Nextcloud, based on the Breeze Dark theme made by the KDE Project
+This is a breeze dark theme for Nextcloud, based on the Breeze Dark theme made by the KDE Project.
 
 ## Usage
 
-After you install the theme, you need to enable it. There a 2 options for this, a global and a personal one.
+After you install the theme, you need to enable it. There a two options for this, a global and a personal one.
 
 ### Global
 
-The global option enables the theme by default for all users and unauthenticated visits. Without this option stuff like the login and sharing pages won't be styled. There is also an option to exclude the login page from being themed while the rest is. You can find the global options here:
+The global option enables the theme by default for all users and unauthenticated visits. Without this option items like the login and sharing page won't be styled. There is also an option to exclude the login page from the theme you set. You can find the global options here:
```

Comment:
> Without this option items like the login and sharing page won't be styled.

I'm not sure going from ```pages``` to ```page``` is correct here as I would consider there to be multiple sharing pages in Nextcloud.

> There is also an option to exclude the login page from the theme you set.

The original sentence was not great so I definitely agree that it should be changed, but I'm not sure about ```theme you set```. To me it implies there are multiple themes that this option can apply to, which isn't the case. The option only applies to Breeze Dark. Maybe it just needs to be trimmed as the end of the sentence was pretty unnecessary. So:
```
There is also an option to exclude the login page from being themed.
```

There is also the option of changing both sentences as they are related but the flow between them isn't great, if that makes sense. Let me know what you think.

And just so we are on the same page, these two:

> Without this option items like the login and sharing page won't be styled. There is also an option to exclude the login page from the theme you set.

Reply:
I made additional corrections to the sentence :wink:.